asistente_ladm_col/tests/test_toolbar.py

```diff
@@ -22,6 +22,7 @@ def setUpClass(cls):
         import_qgis_model_baker()
         import_asistente_ladm_col()  # Import plugin
         cls.app = AppInterface()
+        cls.app.core.initialize_ctm12()  # We need to initialize CTM12
```

Comment:
What if you initialize CTM12 [after this line](https://github.com/SwissTierrasColombia/Asistente-LADM-COL/blob/lev_cat_1_1/asistente_ladm_col/tests/utils.py#L59) instead (and remove any other call to such method in unit tests)?

Reply:
What a good idea, adjustment made. initialize CTRM12 from other classes were removed. Available in commit 513f31a